### PR TITLE
Fix 'data' field not being made available to rust_test rule.

### PR DESCRIPTION
**Pull request by mfarrugi**
Setting data wasn't actually being made available to tests, this fixes that.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by mfarrugi** [comment]
I signed it!

File: rust/rust.bzl [hunk 1]
```diff
@@ -373,6 +373,7 @@ def _rust_test_common(ctx, test_binary):
                              rust_flags = ["--test"])
 
   compile_inputs = (target.srcs +
+		    ctx.files.data +
```
Comment (davidzchen) [inline rust/rust.bzl]: Align this with the other lines.

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by mfarrugi** [comment]
Added the example and noticed it was also missing from `rust_binary`, the test case I was using earlier had been using a data file created by a genrule.

**Comment by davidzchen** [comment]
This change LGTM. Thanks for your contribution.

**Comment by acmcarther** [comment]
I believe this is causing a test failure in OSX

**Comment by acmcarther** [comment]
I believe this is causing a test failure in osx: https://ci.bazel.build/blue/organizations/jenkins/rules_rust/detail/rules_rust/800/pipeline

But I don't know how to trigger a build of master. Creating revert PR to see.

**Pull request merged**